vip-cache-manager/class-wpcom-vip-cache-manager.php

```diff
@@ -456,12 +476,12 @@ public function queue_post_purge( $post_id ) {
 		}
 
 		if ( ! is_post_type_viewable( $post->post_type ) ) {
-			return;
+			return false;
```

Comment:
Here (and below) SonarLint complained that the `return` statement is not used consistently — it should either return nothing everywhere or return a boolean.